dev-python/seaborn/seaborn-0.10.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1 virtualx
```

Comment:
... so you don't need `virtualx` or what?

Reply:
apparently it's no longer needed, yes